### PR TITLE
fingerprint for Taiwan 2020 AURIS/COROLLA SPORT

### PR DESCRIPTION
**Pull request by ceasar0524**
fingerprint for Taiwan 2020 AURIS/COROLLA SPORT
Device: comma two
Dongle ID: 3025b993d0e21cdf

### CODE REVIEW

File: selfdrive/car/toyota/values.py [hunk 1]
```diff
@@ -630,6 +630,7 @@ class CAR:
       b'\x018966312R3100\x00\x00\x00\x00',
       b'\x018966312S5000\x00\x00\x00\x00',
       b'\x018966312S7000\x00\x00\x00\x00',
+      b'\0018966312M0000\000\000\000\000',
```
Comment (pd0wm) [inline selfdrive/car/toyota/values.py:633]:
```suggestion
      b'\x018966312M0000\x00\x00\x00\x00',
```

File: selfdrive/car/toyota/values.py [hunk 2]
```diff
@@ -649,6 +650,7 @@ class CAR:
       b'\x018965B12530\x00\x00\x00\x00\x00\x00',
       b'\x018965B1255000\x00\x00\x00\x00',
       b'8965B12361\x00\x00\x00\x00\x00\x00',
+      b'\0018965B12530\000\000\000\000\000\000',
```
Comment (pd0wm) [inline selfdrive/car/toyota/values.py:653]:
```suggestion
      b'\x018965B12530\x00\x00\x00\x00\x00\x00',
```

File: selfdrive/car/toyota/values.py [hunk 3]
```diff
@@ -664,12 +666,14 @@ class CAR:
       b'\x01F152612B90\x00\x00\x00\x00\x00\x00',
       b'\x01F152612C00\x00\x00\x00\x00\x00\x00',
       b'F152602191\x00\x00\x00\x00\x00\x00',
+      b'\001F152612B51\000\000\000\000\000\000',
```
Comment (pd0wm) [inline selfdrive/car/toyota/values.py:669]:
```suggestion
      b'\x01F152612B51\x00\x00\x00\x00\x00\x00',
```

File: selfdrive/car/toyota/values.py [hunk 4]
```diff
@@ -664,12 +666,14 @@ class CAR:
...
     ],
     (Ecu.fwdRadar, 0x750, 0xf): [
       b'\x018821F3301100\x00\x00\x00\x00',
       b'\x018821F3301200\x00\x00\x00\x00',
       b'\x018821F3301300\x00\x00\x00\x00',
       b'\x018821F3301400\x00\x00\x00\x00',
+      b'\0018821F3301400\000\000\000\000',
```
Comment (pd0wm) [inline selfdrive/car/toyota/values.py:676]:
```suggestion
      b'\x018821F3301400\x00\x00\x00\x00',
```

File: selfdrive/car/toyota/values.py [hunk 5]
```diff
@@ -680,6 +684,7 @@ class CAR:
       b'\x028646F1202000\x00\x00\x00\x008646G2601200\x00\x00\x00\x00',
       b'\x028646F1202100\x00\x00\x00\x008646G2601400\x00\x00\x00\x00',
       b'\x028646F1202200\x00\x00\x00\x008646G2601500\x00\x00\x00\x00',
+      b'\0028646F1202100\000\000\000\0008646G2601400\000\000\000\000',
```
Comment (pd0wm) [inline selfdrive/car/toyota/values.py:687]:
```suggestion
      b'\x028646F1202100\x00\x00\x00\0008646G2601400\x00\x00\x00\x00',
```

**Comment by adeebshihadeh** [comment]
added your fw in 78027b471fdaa0df87f9398d51214caf551f2af0

**Pull request closed without merging**